packages/contracts/contracts/L2/L1Block.sol

```diff
@@ -16,12 +16,14 @@ contract L1Block {
     uint256 public timestamp;
     uint256 public basefee;
     bytes32 public hash;
+    uint256 public seqnr;
 
     function setL1BlockValues(
         uint256 _number,
         uint256 _timestamp,
         uint256 _basefee,
-        bytes32 _hash
+        bytes32 _hash,
+        uint256 _seqnr
```

Comment:
Is it always expected that `seqnr` increments by 1? We could `revert` if it doesn't increment correctly but we also have `number` which we could do that on

Reply:
Got it, you should add that explanation to the specs since that clarifies its usage quite a bit

Reply:
See below review comment on spec doc.